### ISSUE TITLE
turn-off-monitor@zablotski - Does not turn off the monitor

### ISSUE THREAD

**Issue by Padre2** [linuxmint/cinnamon-spices-applets#2719]
* Applet version/Build date - Updated on 2019-10-20 00:00:27 
 * Cinnamon version 4.4
 * Distribution - Arch
 * Graphics hardware - AMD/ATI
 * Snippet or attachment of ~/.cinnamon/glass.log
info t=2019-11-30T14:42:30Z Loaded applet turn-off-monitor@zablotski in 16 ms
 * Snippet or attachment of ~/.xsession-errors
Cjs-Message: 14:42:30.417: JS LOG: [LookingGlass/info] Loaded applet turn-off-monitor@zablotski in 16 ms
 ```
<!-- Author list: https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.github/CODEOWNERS -->

**Notify author of applet**
turn-off-monitor@zablotski

**Issue**
monitor screen goes black for ~ 3 seconds and comes back on.

**Steps to reproduce**
Click icon

**Expected behaviour**
Monitor turns off.

Monitor : LG

**Comment by claudiux** [comment]
@Padre2 After clicking on the icon, do not move the mouse. What's going on, then?

**Comment by Padre2** [comment]
@claudiux
If I position the cursor and hold the mouse absolutely still using my left hand and left-click, it will turn off the display and the D-SUB No signal message will appear and the power led will come on, then the D-SUB message will disappear and the power light will flash. The slightest  mouse movement will open the display again. Can be done with one hand, but need to take extra care not to have any movement.

**Comment by claudiux** [comment]
@Padre2 Please try to edit the `~/.local/share/cinnamon/applets/turn-off-monitor@zablotski/applet.js` script and change the value `1000` by `5000` (line 36). Save this file and restart Cinnamon. Your mouse will become inactive during 5 seconds (5000 ms) instead of 1 second (1000 ms) actually.

If that suits you, I'll modify this applet to implement some settings. Its author seems to be inactive on Github.

**Comment by Padre2** [comment]
@claudiux  Thank you for looking at this. It does reduce the sensitivity making it useable.  However, after further consideration I will probably not use this applet. When I need to blank the screen, I can use  _xset dpms force off_  from an alias and it doesn't even need _sudo._ 
I see you often help with solutions for various applets, which is much appreciated. Is there any way I can contact you directly on this subject, without going through github?

**Comment by claudiux** [comment]
@Padre2 I'm proposing the PR #2729 to improve this applet. Can you test it, please?

The best way to contact me is Github. I watch Github more often than my emails.

Kind regards.
Claudiux

**Comment by Padre2** [comment]
@claudiux  I'm happy to assist with testing.  I am not yet fully familiar with the workings of GitHub. Will you be updating the applet or do I need to download it?

The reason I asked about an email address, is that I have several issues with certain applets and how Cinnamon behaves, e.g. I noticed differences in the way that the CPU Temperature applet displays on different distributions on the same machine, but don't want to go into this on GitHub, at least not yet. Although I use Arch that I compiled, I also have available Manjaro (Arch) and Feren (Ubuntu), both with Cinnamon DE.
